reporting.md

```diff
@@ -114,11 +114,9 @@ We also define the following terms for further use in this document:
 
 * This document is intended as a companion to the specification for SMTP MTA
     Strict Transport Security (MTA-STS, TODO: Add ref).
-* The Public Key Pinning Extension for HTTP [@!RFC7469] contains a JSON-based
-    definition for reporting individual pin validation failures.
-* The Domain-based Message Authentication, Reporting, and Conformance (DMARC)
```

Comment:
Unlike DNSSEC & DANE, these are not directly related to MTA-STS. IMO these are 'good-to-have' but not required to be mentioned.